src/lib/db/migrations/20230310111029_initial.ts

```diff
@@ -20,16 +20,23 @@ export async function up(knex: Knex): Promise<void> {
 
   await knex.schema.createTable('certificate', (def) => {
     def.uuid('id').defaultTo(knex.raw('uuid_generate_v4()'))
-    def.integer('capacity').notNullable().index('capacity_index')
-    def.integer('co2e').notNullable().index('co2e_index')
-    def.string('owner', 48).notNullable()
+    def.integer('hydrogen_quantity_mwh').notNullable().index('hydrogen_quantity_mwh_index')
+    def.integer('embodied_co2').nullable().index('embodied_co2_index').defaultTo(null)
+    def.string('energy_owner', 48).notNullable()
+    def.string('hydrogen_owner', 48).notNullable()
     def
-      .enum('state', ['initialized', 'issued', 'revoked'], {
-        enumName: 'certificate_state',
+      .enu('status', ['initialised', 'issued', 'revoked', 'cancelled'], {
+        useNative: true,
+        enumName: 'certificate_status',
+      })
+      .defaultTo('initialised')
+    def
+      .enum('state', ['pending', 'created'], {
```

Comment:
I think `created` should be `initiated` to match the API verb

Reply:
Updated and removed status.